docker-compose.yml

```diff
@@ -11,8 +11,10 @@ services:
             - python
             - python-sync
             - redis
+            - cherami
         environment:
-            - WAIT_FOR=go,node,python,python-sync
+            - WAIT_FOR=go,node,python,python-sync,cherami
```

Comment:
@breerly Question: How do I wait for cherami docker to start(which will take 40s) before running any test? I added 'cherami' here but seems it doesn't work. Does this require cherami server to accept the 8080 ping request?